api/db-cdr/csv/db_domain.csv

```diff
@@ -1,16 +1,16 @@
 "domain_id","domain_display","domain_desc","db_type","domain_route",concept_id,count_value
-"Condition","Conditions","Conditions are records of a Person suggesting the presence of a disease or medical condition stated as a diagnosis, a sign or a symptom, which is either observed by a Provider or reported by the patient.","domain","condition",19,0
+"Condition","Diagnoses","Conditions are records of a Person suggesting the presence of a disease or medical condition stated as a diagnosis, a sign or a symptom, which is either observed by a Provider or reported by the patient.","domain_filter","condition",19,0
```

Comment:
I think set concept_id to null where it is zero in these csvs. Then you you can do findByDbTypeAndConceptIdIsNotNull . Be a little cleaner.

Reply:
Changed.

Reply:
Changed concept id to null where it was 0.